io/net/multiport_stress.py.data/multiport_stress.yaml

```diff
@@ -2,4 +2,25 @@ host_interfaces: "enP4p1s0f1,enP4p1s0f2"
 peer_ips: "100.10.10.12,200.20.20.22"
 count: "1100"
 host_ips: ""
-netmask: "" 
+netmask: ""
+peer_user: ""
```

Comment:
can we have start practise please.

in some of the yaml,  peer_user and peer_password comes above host_ips.

Eventhough it is not functional related.  It helps user to edit them faster.